introduction.md

```diff
@@ -0,0 +1 @@
+Kon'nichiwa! My name is Alberto, I'm a Web Dev and my favourite codeschool path is Ruby
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.